participants/DBerke/buggy_function.py

```diff
@@ -20,7 +20,7 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
     if math.floor(decimals) != decimals:
         raise OSError('decimals should be an integer!')
 
-    hours_num = angle_in_degrees*24/180
+    hours_num = angle_in_degrees*24/360
```

Comment:
This looks good, although to keep with PEP8 there should be spaces between `*` and `/`.